driver/src/main/java/org/kaazing/k3po/driver/internal/netty/bootstrap/http/HttpChildChannelSource.java

```diff
@@ -125,9 +126,7 @@ public void channelClosed(ChannelHandlerContext ctx, ChannelStateEvent e) throws
             case CONTENT_COMPLETE:
                 break;
             default:
-                ChannelException exception = new ChannelException("Channel closed unexpectedly");
-                exception.fillInStackTrace();
-                fireExceptionCaught(httpChildChannel, exception);
+                Channels.fireChannelAborted(httpChildChannel);
```

Comment:
Why is this only done in the write status switch statement and not the switch on the read status (which still throws the "Channel closed unexpectedly" exception} ?